[Bug] Error at testing flag_embedding_reranker

```python
    @passage_reranker_node
    def flag_embedding_reranker(queries: List[str], contents_list: List[List[str]],
                                scores_list: List[List[float]], ids_list: List[List[str]],
                                top_k: int, batch: int = 64, use_fp16: bool = False,
                                model_name: str = "BAAI/bge-reranker-large",
                                ) -> Tuple[List[List[str]], List[List[str]], List[List[float]]]:
        """
        Rerank a list of contents based on their relevance to a query using BAAI Reranker model.
    
        :param queries: The list of queries to use for reranking
        :param contents_list: The list of lists of contents to rerank
        :param scores_list: The list of lists of scores retrieved from the initial ranking
        :param ids_list: The list of lists of ids retrieved from the initial ranking
        :param top_k: The number of passages to be retrieved
        :param batch: The number of queries to be processed in a batch
        :param use_fp16: Whether to use fp16 for inference
        :param model_name: The name of the BAAI Reranker model name.
            Default is "BAAI/bge-reranker-large"
        :return: tuple of lists containing the reranked contents, ids, and scores
        """
        device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
>       model = FlagReranker(
            model_name, use_fp16=use_fp16, device=device
        )
E       TypeError: __init__() got an unexpected keyword argument 'device'

../../../../autorag/nodes/passagereranker/flag_embedding.py:32: TypeError
```

I am using FlagEmbedding==1.2.8 and m2 pro
Please fix this error ASAP. After that, I can release 0.1.2 version.

--- a/autorag/nodes/passagereranker/flag_embedding.py
+++ b/autorag/nodes/passagereranker/flag_embedding.py
@@ -28,9 +28,8 @@ def flag_embedding_reranker(queries: List[str], contents_list: List[List[str]],
         Default is "BAAI/bge-reranker-large"
     :return: tuple of lists containing the reranked contents, ids, and scores
     """
-    device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
     model = FlagReranker(
-        model_name, use_fp16=use_fp16, device=device
+        model_name_or_path=model_name, use_fp16=use_fp16
     )
     tasks = [flag_embedding_reranker_pure(query, contents, scores, top_k, ids, model)
              for query, contents, scores, ids in zip(queries, contents_list, scores_list, ids_list)]

